xds/src/main/java/io/grpc/xds/BootstrapperImpl.java

```diff
@@ -52,6 +52,12 @@
   private static final String BOOTSTRAP_PATH_SYS_PROPERTY = "io.grpc.xds.bootstrap";
   @VisibleForTesting
   static String bootstrapPathFromSysProp = System.getProperty(BOOTSTRAP_PATH_SYS_PROPERTY);
+  private static final String BOOTSTRAP_CONFIG_SYS_ENV_VAR = "GRPC_XDS_BOOTSTRAP_CONFIG";
+  @VisibleForTesting
+  static String bootstrapConfigFromEnvVar = System.getenv(BOOTSTRAP_CONFIG_SYS_ENV_VAR);
+  private static final String BOOTSTRAP_CONFIG_SYS_PROPERTY_VAR = "io.grpc.xds.bootstrap_value";
```

Comment:
Why not "io.grpc.xds.bootstrap.config"?

Reply:
Well, this is given by the design doc. https://docs.google.com/document/d/1QR5Gn8NZXOcZU83mPI-1BWkB1y_wtdnFYIY-UdwIZbM/edit?resourcekey=0-2MYNzBwbCb5adftxfd75JA

Reply:
Okay...

Reply:
bootstrap_value sounds fine to me, with it "in" the io.grpc.xds package. Although I think we'd generally spell it "bootstrapValue", not with an underscore. If we used "bootstrap.file" and "bootstrap.file", dots would seem fair, but we don't.

Reply:
Let's just use `"io.grpc.xds.bootstrapValue"`. This is Java-specific, it could be up to our own decision.